src/main/java/Duke.java

```diff
@@ -1,10 +1,93 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
+        String lineBreak = "---------------------------------------------";
+        String line = "";
+        Task[] list = new Task[100];
+        int index = 0;
+        Scanner in = new Scanner(System.in);
+        /*String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello from\n" + logo);*/
```

Comment:
Since dead code can be retrieved using revision control, consider deleting it instead of just commenting it out